SU-SWS/decanter
# Main Nav Light on Dark Background Color Variant

# READY FOR REVIEW

# Summary
- A variant of main nav to be used on dark backgrounds, e.g., the Decanter site
- The only difference between this variant and the default are the first level links on desktop, and the mobile hamburger button. The dropdown/mobile version are the same as the default styles.
- I was able to show it to @JohnHolleman before he left for long weekend. It has his blessing 😄 

# Needed By (Date)
- Maybe before v5 release

# Urgency
- N/A

# Steps to Test

1. Pull this branch and run `grunt styleguide`
2. Look at the .su-main-nav--dark variant and see if the look and behavior are ok

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- https://github.com/SU-SWS/decanter/issues/328
- https://github.com/SU-SWS/decanter/issues/333
